### ISSUE TITLE
question regarding development

### ISSUE THREAD

**Issue by imnvr** [KevinTyrrell/FishingBot#6]
Hello, do you plan to keep this under active development ? There is no other FOSS tool that does this and it would be great if we had something like this actively developed. It has a lot of potential. Just my 2 cents.

**Comment by KevinTyrrell** [comment]
Hey man, 

Yes I do, definitely. This project will always be something I come back to and update. (That should tell you how good World of Warcraft is as a game! haha) Lately I've been doing more coding in the C programming language but I will come back and continue to update this. One thing I wanted is to make the UI as simple as possible so adding new features is something I don't want because it will clutter the user interface. If you have a good idea though I'll be happy to implement it.

I'm really honored that so many users enjoy my program. Originally I just made this for myself/for fun and I'm so happy many people enjoy using it.

**Comment by Jaladhjin** [comment]
Would this theoretically work on most versions of English US WoW ?

For example live retail Legion ?

I saw it mentioned SuperMacro was strongly encouraged for like 1.12.

**Comment by KevinTyrrell** [comment]
@Jaladhjin 

I don't have an active retail account but I did test it back when Warlords of Draenor was the current expansion and here's what I noticed.

- The amount of time that each fishing cast takes is a lot lower than before so the speed of the bobber movement needs to be set fast.
- I had my in game graphical settings set to max, but for some reason the animation of the fish and the water that sprays was way less than in Vanilla through Wrath of the Lich King. This likely means that sensitivity in my program should be turned way up so it can detect the very subtle splash. 
- The default tooltip color looks slightly different than before which actually would break the program from working. Just make sure that calibration works and if it does then you are good. The location of the tooltip may have changed so you can right click on the calibration button to select a custom search area.

If you try it, please let me know how it performs. I would be very grateful.

**Comment by mtoedtem** [comment]
Hello Kevin,

Love your tool. Works like a charm. The current version of classic doesn't allow scripts thus the lure logic is broken. 

Would you be able to update it to instead press 1 for fishing and 2 for lure?

Thanks for the great work.

**Comment by KevinTyrrell** [comment]
@mtoedtem You're correct. I noticed this issue when WoW Classic launched. It is on my radar to be added, however something you can do in the meantime is to install the addon called FishingBuddy. Enable its automatic lure feature and that should handle the lure problem for now.
